cmd/ddev/cmd/root.go

```diff
@@ -190,3 +197,64 @@ func checkDdevVersionAndOptInInstrumentation(skipConfirmation bool) error {
 
 	return nil
 }
+
+func suggestAddonForUnknownCommand(command string) {
+	if os.Getenv("DDEV_NONINTERACTIVE") != "" {
+		return
+	}
+	app, err := ddevapp.GetActiveApp("")
+	if err != nil {
+		return
+	}
+	// TODO: cache this
+	repos, err := ddevapp.ListAvailableAddons(true)
+	if err != nil {
+		return
+	}
+	var matches []string
+	for _, repo := range repos {
+		if fuzzy.Match(command, *repo.Name) {
+			matches = append(matches, *repo.FullName)
+		}
+	}
+	if len(matches) < 1 {
+		return
+	}
+	selectedAddon := matches[0]
+	// TODO: add check if add-on is already installed
+	if len(matches) > 1 {
+		// TODO: make the output fancier
+		for i, match := range matches {
+			fmt.Printf("%d. %s\n", i+1, match)
+		}
+		selection := util.Prompt("Do you want to install one of these add-ons?", "choose a number")
+		choice, err := strconv.Atoi(selection)
+		if err != nil || choice < 1 || choice > len(selection) {
+			util.Failed("Invalid selection")
+		}
+		selectedAddon = matches[choice-1]
+	} else {
+		y := util.Confirm(fmt.Sprintf("Do you want to install %s add-on?", selectedAddon))
```

Comment:
```suggestion
		y := util.Confirm(fmt.Sprintf("Do you want to install '%s' add-on?", selectedAddon))
```

Make the addon name stand out a little more. This will aid the developer if they later choose to manually install it.